### PR TITLE
[CJP-3314] Tombstone release

### PR DESCRIPTION
**Pull request by stephenc**
[CJP-3314](https://cloudbees.atlassian.net/browse/CJP-3314)

- Actually needs to do some work though, removing the old update center entry

@reviewbybees

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -34,24 +34,13 @@
   </parent>
```
Comment (rsandell) [inline pom.xml]: I guess this needs an updated maven release plugin since core dep is 1.509?

Comment (stephenc) [inline pom.xml]: Yep

Comment (stephenc) [inline pom.xml]: or just remember to run `mvn org.apache.maven.plugins:maven-release-plugin:2.5.2:prepare org.apache.maven.plugins:maven-release-plugin:2.5.2:perform -B`

Comment (rsandell) [inline pom.xml]: And who remembers to do that nowadays ;)

**Comment by rsandell** [comment]
:bee:

File: pom.xml (hunk 1 above)
Comment (stephenc) [inline pom.xml]: true... hence b35a32f

**Comment by jglick** [comment]
:bee:

**Comment by oleg-nenashev** [comment]
:bee:

**Comment by stephenc** [comment]
@reviewbybees done

**Pull request merged**